### ISSUE TITLE
Add support for Angular 15

### ISSUE THREAD

**Issue by plaskowski-solwit** [intbot/ng2-pdfjs-viewer#197]
Hi! 

Please update peer deps to support Angular 15 and create new release. 

Regards

**Comment by angelthebault** [comment]
That is to prevent npm from crashing on the command install with the following error :

` npm` ERR! ERESOLVE unable to resolve dependency tree

A big thank you for the work on this library. Very easy to use and yet so powerfull !

**Comment by alessandro-vincelli** [comment]
Is this project still alive?

**Comment by codehippie1** [comment]
@OliverGrack @alessandro-vincelli The entire project is upgraded to Angular 15. All PRS are merged. I also had to upgrade SampleApp to test if the upgrade once. Sorry that it took a bit time. I got occupied with other projects and the holidays.

The new package (V 15.0) is available in npm
@OliverGrack - Yes, I could certainly use some help. I will add you to the team on github, where you get authority to merge PRs. Sample app can be used for testing with a tar file. I just added a autodeploy.ps1 file to the repo, all it needs is to run, it will spin up Sample app, move through folders as necessary and producing a testing environment with a 'tgz' file.
